app/subscriber/src/components/media-type-filters/utils/determineStore.ts

```diff
@@ -0,0 +1,12 @@
+import { IContentState } from 'store/slices';
```

Comment:
When passing a store to the `MediaTypeFilter` this util will determine the appropriate `storeFilter` method to call. This reduces the number of props we have to pass to the filter.